spec/spec.md

```diff
@@ -202,15 +201,15 @@ sequenceDiagram
 - To bind an owner of a DID to a controller of a certain origin, a Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].
 - For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MUST be discovered using either DID Relative URLs stored in an HTTPS URL or ID Hub be used in combination with Identity Hubs as defined in [[def: Identity Hub (0.0.1 Predraft)]] (Decentralized Web Node v0.0.1 predraft).
 
-This profile uses certain versions of specifications that have not yet reached final status: For more details see Normative References section.
+This profile uses certain versions of specifications that have not yet reached final status: For more details see [Normative References](#normative-references) section.
 
   - First Implementer's Draft of Self-Issued OpenID Provider v2 specification
   - First Implementer's Draft of OpenID for Verifiable Prensetations specification
-  - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). We will continue to use the term ID Hub rather than Decentralized Web Node to avoid confusion.
+  - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). Throughout the document, the term ID Hub, rather than Decentralized Web Node, will be used to avoid confusion.
 
-#### Security Considerations
+This JWT VC Presentation Profile currently only supports response mode `direct_post`, sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.
```

Comment:
```suggestion
The JWT VC Presentation Profile currently only supports response mode `direct_post` defined in [[ref: OpenID4VP ID1]], sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.
```